src/SearchButton.cc

```
@@ -51,4 +54,14 @@ void SearchButton::paintIcon(QPainter *painter, const QRectF &iconRect, const qr
     painter->drawLine(handleStart, handleEnd);
 }
 
+void SearchButton::mousePressEvent(QMouseEvent *event)
```

Clang (14.0.6)

[warning] method 'mousePressEvent' can be made static

(readability-convert-member-functions-to-static)